website/public/js/controllers/inboxCtrl.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+habitrpg.controller("InboxCtrl", ['$scope', '$rootScope',
+    function($scope, $rootScope) {
+      $scope.copyToDo = function(message) {
+        var taskNotes = env.t("taskFromInbox",  {
+          from: message.uuid == 'system'
```

Comment:
It's confusing to have a ternary here. I think this logic should be outside of the object.